spring-kafka/src/main/java/org/springframework/kafka/listener/KafkaMessageListenerContainer.java

```diff
@@ -1937,12 +1937,8 @@ else if (this.syncCommits) {
 
 		private void commitAsync(Map<TopicPartition, OffsetAndMetadata> commits, int retries) {
```

Comment:
Looks like `retries` param is out of use now...
Since we don't retry async commit any more, I guess it is time to let this `retries` param go...

Reply:
Doh - yes - I meant to clean that up.